go-controller/pkg/types/const.go

```diff
@@ -94,6 +94,7 @@ const (
 	DefaultNoRereoutePriority             = 102
 	EgressSVCReroutePriority              = 101
 	EgressIPReroutePriority               = 100
+	EgressIPRerouteQoSRulePriority        = 103
```

Comment:
> what made you pick 103 here? Does this conflict at all with EgressQoS also applied to the pod? 

no conflicts, 103 is a random number and the only reason I use a unique number is because I made the priority a "key" in the dbIndex for this QoS Rule. so its nice for me to get predicate based on priority and I know its an EIP owned one not EQoS owned one (although I also give it a unique name/family id keys)